GitExtUtils/GitUI/ToolStripExSystemRenderer.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace GitUI
 {
-    internal sealed class ToolStripExRenderer : ToolStripSystemRenderer
+    public sealed class ToolStripExSystemRenderer : ToolStripSystemRenderer
```

Comment:
Does it need to be public?

Reply:
It became public so it could be [called](https://github.com/gitextensions/gitextensions/pull/9795/files#diff-81d35d9561cd05f67e9b83ae32c98f4398fcff5e2377227227e0b77d207f5066R17) it in `GitUI/UserControls/ToolStripEx.cs` construstor